config/main.py

```diff
@@ -1500,7 +1500,8 @@ def add_portchannel(ctx, portchannel_name, min_links, fallback):
         ctx.fail("{} already exists!".format(portchannel_name))
 
     fvs = {'admin_status': 'up',
-           'mtu': '9100'}
+           'mtu': '9100',
+           'lacp_key': 'auto'}
```

Comment:
can we read the `lacp_key` from the `cli` in case one would like to give specific key instead of auto generating it?

Reply:
lacp_key attribute was not exposed to the user at all and was always set to zero.
in the scope of fixing this issue, we are now setting the lacp_key to a unique ID (xxxx) derived from the Portchannelxxxx.
and this is what the auto mode is doing.

I don't think adding this CLI is part of the scope of this bug fix, it is another feature that we could provide, but we need to fix the bug that started this first (https://github.com/Azure/sonic-buildimage/issues/4009)
This issue was marked as closed, but until this PR will be merged we still haven't fixed the bug, and it still an issue in the system.
If needed we will create a new PR with this improvement.